File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/EntitiesListWithSearchFragment.java
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist;

/**
 * @author Vadim Boitsov <vadimboitsov1@gmail.com>
 */

public interface EntitiesListWithSearchFragment extends EntitiesListFragment {

    String getSearchQuery();

    void switchToNormalMode();

    void switchToSearchMode();
}

File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/EntitiesListWithSearchFragment.kt
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist

/**
 * @author Vadim Boitsov <vadimboitsov1></vadimboitsov1>@gmail.com>
 */
interface EntitiesListWithSearchFragment : EntitiesListFragment {

    fun getSearchQuery(): String

    fun switchToNormalMode()

    fun switchToSearchMode()
}
